### PR TITLE
chore: Pricing page url updated

### PR DESCRIPTION
**Pull request by amithire1201**
Pricing page URL updated to https://wpastra.com/starter-templates-plans/

### CODE REVIEW

File: inc/classes/class-astra-sites.php [hunk 1]
```diff
@@ -743,6 +743,12 @@ public function create_image() {
 				if ( did_action( 'elementor/loaded' ) ) {
 					$result['data'] = Astra_Sites_Elementor_Images::get_instance()->get_attachment_data( $image );
 				}
+				if ( 0 === $photo_id ) {
+					/**
+					 * This flag ensures these files are deleted in the Reset Process.
+					 */
+					update_post_meta( $image, '_astra_sites_imported_post', true );
+				}
```
Comment (maheshwaghmare) [inline inc/classes/class-astra-sites.php:751]: Do we need about change?

File: inc/classes/class-astra-sites.php [hunk 2]
```diff
@@ -759,9 +765,6 @@ public function create_image() {
 				update_option( 'astra-sites-saved-images', $saved_images, 'no' );
 			}
 
-			$saved_images[] = $photo_id;
-			update_option( 'astra-sites-saved-images', $saved_images, 'no' );
-
```
Comment (maheshwaghmare) [inline inc/classes/class-astra-sites.php:764]: Is this from IST? or we need this change?

**Review by maheshwaghmare — CHANGES_REQUESTED** [review]

File: inc/classes/class-astra-sites.php (hunk 1 above)
Comment (amithire1201) [reply inc/classes/class-astra-sites.php]: Unwanted changes reverted.

File: inc/classes/class-astra-sites.php (hunk 2 above)
Comment (amithire1201) [reply inc/classes/class-astra-sites.php]: Unwanted changes reverted.

**Review by maheshwaghmare — APPROVED** [review]

**Pull request merged**